src/Controller/AddOnOptionsController.py

```diff
@@ -549,4 +573,5 @@ def __init__(self, window):
 
     def show_add_on_options(self):
         self.options_window = AddOnOptions(self.window)
-        self.options_window.show()
+        self.options_window.directory_updated.connect(lambda new_dir: self.window.directory_updated.emit(new_dir))
```

Comment:
Limits of character past 79

Reply:
Is that the project defined limit?
Linus Torvald has said that terminals are no longer 80 chars and 79 is overly restrictive.
I review from my phone, so super long lines are hard to read even in landscape mode.
For reformatting existing code, I tend to get a histogram and pick a number that seems like a reasonable cut off without having a huge fraction of the code reformatted.

Reply:
For now i am following the Onko Naming Conventions and so this comment is not definite. We are currently asking ashley if he is strict on the length and comment formats, etc. My current comments on this pull request will just be reminders for when our team makes the pull request. All comments will be deleted afterwards